lib/raven/processor/sanitizedata.rb

```diff
@@ -5,6 +5,7 @@ class Processor::SanitizeData < Processor
     INT_MASK = 0
     DEFAULT_FIELDS = %w(authorization password passwd secret ssn social(.*)?sec)
     CREDIT_CARD_RE = /^(?:\d[ -]*?){13,16}$/
+    REGEX_SPECIAL_CHARACTERS = %w(. $ ^ { [ ( | ) * + ?)
```

Comment:
alternate suggestion: allow regexes directly in `DEFAULT_FIELDS | sanitize_fields`. That way we don't need to guess (and you can still keep backwards compat).